src/App.test.tsx

```diff
@@ -1,8 +1,10 @@
-import { render, screen } from '@testing-library/react';
+import { render } from '@testing-library/react';
 import App from './App';
 
-test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
+describe('<App />', () => {
```

Comment:
How about just `App`?
```suggestion
describe('App', () => {
```

Reply:
I saw that in some training and I kind of liked the message it carries: "describe the App component" (while `describe('App')` carries a less clear message "describe the App" — the whole application?).

But it does not change much to let's go for `App` alone ;-)

Reply:
✅ Done in 39a594b :)